test/Buyer/viewFarmOfferings.test.js

```diff
@@ -0,0 +1,15 @@
+import { describe } from "mocha";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
import { describe } from 'mocha';
```